src/includes/getting-started-install/react-native.mdx

```diff
@@ -0,0 +1,66 @@
+Add the `@sentry/react-native` dependency:
+
+```bash {tabTitle:npm}
+npm install --save @sentry/react-native
+```
+
+```bash {tabTitle:Yarn}
+yarn add @sentry/react-native
+```
+
+<Note>
+
+If you are using Expo, see [How to Add Sentry to Your Expo Project](https://docs.expo.io/guides/using-sentry/). This SDK works only for either ejected projects or projects that directly use React Native.
```

Comment:
```suggestion
If you are using Expo, see [How to Add Sentry to Your Expo Project](https://docs.expo.io/guides/using-sentry/). This SDK works for both managed and bare projects.
```
Original text is no longer true as Expo updated it so it works on all expo projects now